libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -16,7 +16,7 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @Param: TYPE
     // @DisplayName: Fence Type
     // @Description: Enabled fence types held as bitmask
-    // @Values: 0:None,1:Altitude,2:Circle,3:Altitude and Circle,4:Polygon,5:Altitude and Polygon,6:Circle and Polygon,8:Beacon,9:Altitude and Beacon,10:Circle and Beacon,12:Polygon and Beacon,15:All
+    // @Values: 0:None,1:Altitude,2:Circle,3:Altitude and Circle,4:Polygon,5:Altitude and Polygon,6:Circle and Polygon,7:Altitude, Circle and Polygon,8:Beacon,9:Altitude and Beacon,10:Circle and Beacon,11:Altitude, Circle and Beacon,12:Polygon and Beacon,13:Altitude, Polygon and Beacon,14:Circle, Polygon and Beacon,15:All
```

Comment:
@OXINARF , Although It is waiting for @rmackay9 's comment, remove commas from value for the present.